apps/openmw/mwsound/openal_output.hpp

```diff
@@ -20,6 +25,7 @@ namespace MWSound
     class OpenAL_Output : public Sound_Output
     {
         const DecoderProvider* mDecoderProvider;
+        const VFS::Manager* mVFS;
```

Comment:
This is redundant since the decoder also has it. It also creates an implicit dependency of sorts between the output and the decoder... the output checks its VFS to see if the file exists, then passes that filename to the decoder expecting it to open with its VFS. Not that they're expected to be different, but it's something that can trip up someone casually looking through the code.